docs/guides/auth.md

```diff
@@ -326,3 +343,31 @@ $opts = [
 ];
 $client = new helloworld\GreeterClient('greeter.googleapis.com', $opts);
 ```
+
+###Authenticating with Google (Python)
+
+####Base case - No encryption/authentication
+```python
+channel = implementations.insecure_channel('localhost', 50051)
+stub = helloworld_pb2.beta_create_Greeter_stub(channel)
+...
+```
+
+###Authenticate using OAuth2 token (legacy approach)
```

Comment:
Sorry. Do you mean that JWT access was never implemented in the python oauth2client? Or is it the default application credentials that was never hooked up with it for gRPC?